### PR TITLE
boards/nucleo32: enable SPI and deactivate LED0 auto init by default

### PR DESCRIPTION
**Pull request by aabadie**
Since this is the same problem with the on-board LED being shared with the SPI clock.

### CODE REVIEW

File: boards/nucleo32-f031/include/periph_conf.h [hunk 1]
```diff
@@ -124,6 +124,45 @@ static const pwm_conf_t pwm_config[] = {
...
+        1   /* -> 12000000Hz */
+    }
+};
+
+static const spi_conf_t spi_config[] = {
+    {
+        .dev      = SPI1,
+        .mosi_pin = GPIO_PIN(PORT_B, 5),
+        .miso_pin = GPIO_PIN(PORT_B, 4),
+        .sclk_pin = GPIO_PIN(PORT_B, 3),
```
Comment (vincent-d) [inline boards/nucleo32-f031/include/periph_conf.h]: cs should be defined as well: `.cs = GPIO_UNDEF`

File: boards/nucleo32-f042/include/periph_conf.h [hunk 2]
```diff
@@ -133,6 +133,44 @@ static const pwm_conf_t pwm_config[] = {
...
+        1   /* -> 12000000Hz */
+    }
+};
+
+static const spi_conf_t spi_config[] = {
+    {
+        .dev      = SPI1,
+        .mosi_pin = GPIO_PIN(PORT_B, 5),
+        .miso_pin = GPIO_PIN(PORT_B, 4),
+        .sclk_pin = GPIO_PIN(PORT_B, 3),
```
Comment (vincent-d) [inline boards/nucleo32-f042/include/periph_conf.h]: `.cs = GPIO_UNDEF`

File: boards/nucleo32-f303/include/periph_conf.h [hunk 3]
```diff
@@ -128,6 +128,45 @@ static const pwm_conf_t pwm_config[] = {
...
+        2   /* -> 8000000Hz */
+    }
+};
+
+static const spi_conf_t spi_config[] = {
+    {
+        .dev      = SPI1,
+        .mosi_pin = GPIO_PIN(PORT_B, 5),
+        .miso_pin = GPIO_PIN(PORT_B, 4),
+        .sclk_pin = GPIO_PIN(PORT_B, 3),
```
Comment (vincent-d) [inline boards/nucleo32-f303/include/periph_conf.h]: `.cs = GPIO_UNDEF`

File: boards/nucleo32-f031/include/periph_conf.h (hunk 1 above)
Comment (aabadie) [inline boards/nucleo32-f031/include/periph_conf.h]: Thanks. Fixed !

**Comment by aabadie** [comment]
ping @vincent-d, it should be ok now. Can you ACK ?

**Comment by vincent-d** [comment]
ACK, go when Murdock is happy

**Comment by aabadie** [comment]
Ok, looks like I have to add some of those board in insufficient memory tests....

**Comment by aabadie** [comment]
Smells good @vincent-d, Murdock2 is green!

**Comment by aabadie** [comment]
All green, go !

**Pull request merged**